internal/build/src/tasks/types-definitions.ts

```diff
@@ -1,15 +1,78 @@
 import path from 'path'
 import { readFile, writeFile } from 'fs/promises'
+import ts from 'typescript'
 import glob from 'fast-glob'
 import { copy, remove } from 'fs-extra'
-import { buildOutput } from '@element-plus/build-utils'
+import { Extractor, ExtractorConfig } from '@microsoft/api-extractor'
+import { buildOutput, epRoot, projRoot } from '@element-plus/build-utils'
 import { pathRewriter, run } from '../utils'
 
 export const generateTypesDefinitions = async () => {
+  const typesDir = path.join(buildOutput, 'types', 'packages')
+  const entryDir = path.join(typesDir, 'element-plus')
+  const entryFilePath = path.join(entryDir, 'index.d.ts')
+  const bundlePath = path.join(typesDir, 'element-plus.d.ts')
```

Comment:
Why do we need to change the output file name here? Maybe we can just use the original `index.d.ts`.